fili-wikipedia-example/src/main/java/com/yahoo/wiki/webservice/data/config/names/WikiApiDimensionName.java

```diff
@@ -38,4 +49,16 @@
     public String asName() {
         return camelName;
     }
+
+    public String getDescription() {
```

Comment:
If this thing is a `DimensionName` object, why are we stuffing all of these other parameters in there as well? They don't seem to have anything to do with names.